### PR TITLE
Add new site in Hugo

### PR DESCRIPTION
**Pull request by martina-if**
This new site will contain more documentation and examples

### CODE REVIEW

File: site/content/introduction/01-introduction.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+---
+title: "Introduction"
+weight: 10
+---
+
+# `eksctl` - a CLI for Amazon EKS
```
Comment (errordeveloper) [inline site/content/introduction/01-introduction.md]: The top part of it seems to be the same as the home page... something that we will need to address.

**Comment by errordeveloper** [comment]
@martina-if do you have a preview URL or instruction for someone to build it locally?

File: site/content/introduction/01-introduction.md (hunk 1 above)
Comment (martina-if) [reply site/content/introduction/01-introduction.md]: yes, I'm not sure what do do between those two. What do you think?

File: site/content/usage/09-iam-policies.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      withAddonPolicies:
+        autoScaler: true
+        imageBuilder: true
+```
+*Important*: if a nodegroup includes the `attachPolicyARNs` it must also include the default node policies, like in 
+this example (`AmazonEKSWorkerNodePolicy` and `AmazonEKS_CNI_Policy`).
+
+[comment]: <> (TODO find better example and explain more)
+
+## Managing IAM Users and Roles
```
[the remark this answers is not in the document]
Comment (martina-if) [reply site/content/usage/09-iam-policies.md]: yes, I forgot, thanks

File: site/content/community/02-roadmap.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Road Map"
```
[the remark this answers is not in the document]
Comment (martina-if) [reply site/content/community/02-roadmap.md]: I think it's actually two words

**Comment by martina-if** [comment]
This is a preview from my fork: https://confident-sammet-4a692a.netlify.com/

To build it locally: 
  - install Hugo v0.55.6
  - `cd site/`
  - `hugo serve -D`
  - visit `http://localhost:1313`

I may have forgotten some step but I think that was basically it. Let me know if it doesn't work.

**Comment by gemagomez** [comment]
LGTM, let's merge and work on the next improvements afterwards

**Comment by kalbir** [comment]
For me page renders oddly on mobile -- specifically on Firefox and Chrome on Android. The words "eksctl" "create" & "cluster" are cascading down the page.

If there's an easy fix might be nice to get that sorted.

**Comment by bia** [comment]
Good catch @kalbir! That's coming from some hack-y css on my part. I'll look into it now.

**Comment by bia** [comment]
Most mobile platforms are good now,
Screenshot 2019-06-13 at 14 23 24

except for iPhone 5 :(

Screenshot 2019-06-13 at 14 31 26

**Pull request merged**